packages/twenty-server/src/modules/person/standard-objects/person.workspace-entity.ts

```diff
@@ -44,11 +44,13 @@ import { TimelineActivityWorkspaceEntity } from 'src/modules/timeline/standard-o
 
 const NAME_FIELD_NAME = 'name';
 const EMAILS_FIELD_NAME = 'emails';
+const PHONES_FIELD_NAME = 'phones';
 const JOB_TITLE_FIELD_NAME = 'jobTitle';
 
 export const SEARCH_FIELDS_FOR_PERSON: FieldTypeAndNameMetadata[] = [
   { name: NAME_FIELD_NAME, type: FieldMetadataType.FULL_NAME },
   { name: EMAILS_FIELD_NAME, type: FieldMetadataType.EMAILS },
+  { name: PHONES_FIELD_NAME, type: FieldMetadataType.PHONES },
```

Comment:
I never realized phones weren't indexed at all! I thought it was a formatting issue but it's worse than I thought!